example.py

```diff
@@ -1,5 +1,6 @@
 from pyowlet.PyOwlet import PyOwlet
 import sys
+import time
 from pprint import pprint
 
 if len(sys.argv) != 3:
```

Comment:
Line 7 needs to be "example.py" not "test.py"

Reply:
Good catch! I'll update